library/ix-dev/charts/ix-chart/questions.yaml

```diff
@@ -487,6 +487,14 @@ questions:
                 schema:
                   type: path
                   required: true
+              - variable: sizeLimit
```

Comment:
Let's add a reasonable default?

Reply:
I think 2Gi is okay for most systems?

Reply:
Maybe 1Gi?

Reply:
Probably even less then that might be better..